pages/apim/3.x/event-native-apim/event-native-apim-documentation.adoc

```diff
@@ -0,0 +1,145 @@
+[[event-native-apim-documentation]]
+= Documentation
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/event_native_apim_documentation.html
+:page-folder: apim/event-native-apim
+:page-layout: apim3x
+
+[label label-version]#New in version 3.20.0#
+[label label-version]#BETA release#
+
+== Overview
+
+_This page is for the draft event-native APIM docmentation that Andrew is writing for 3.20. It is here only for initial draft and review purposes. It will be split into different pages and (probably) moved to a new location later._
+
+== Asynchronous APIs
+
+In this release, you can use the gateway to connect to asynchrous APIs. You can configure asynchronous API connections using the RESTful interface to the management API.
+
+The OpenAPI speccification for the management API is available at link:/apim/3.x/management-api/3.19/swagger.json[`swagger.json`] (link:/apim/3.x/apim_installguide_rest_apis_documentation.html#apim_console_api_reference[reference documentation]). Examples of how to use the management API to configure asynchrous APIs can be found in the link:https://github.com/gravitee-io/postman-collections[Gravitee Postman Collections], which are described in link:event_native_apim_using.html[Using Event-native API Management].
+
+The following entrypoints are available.
+
+* SSE
+* Websockets
+* HTTP GET
+* HTTP POST
+
+The following endpoints are available
+
+* Kafka
+* MQQT
+
+== Create a v4 async API 
+
+The link:https://github.com/gravitee-io/postman-collections[Gravitee Postman Collections] contain several examples of how to create an API.
```

Comment:
This Github repository is not public for now. Should we consider moving that one from private to public ?

Reply:
Yes, this will be made available on Postman and the link has been updated.